test/integration/default/bats/snapraid_installed.bats

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bats
 
 @test "snapraid binary is found in PATH" {
-  run which snapraid
+  run sudo su - $SUDO_USER -c "which snapraid"
```

Comment:
Well the reason it was failing previously was because bats was already running under sudo, but in CentOS sudo does not preserve your environment variables (meaning it can't find snapraid). The command above remedies that by running the shell as the SUDO_USER (in this case vagrant) and thereby restoring the environment variables.
Works quite well for our purposes though :)